torch/csrc/jit/frontend/tree_views.h

```diff
@@ -1262,7 +1262,11 @@ inline Expr pep604union_to_union(const Expr& expr) {
       expr.range(),
       Var::create(expr.range(), Ident::create(expr.range(), "Union")),
       List<Expr>::create(expr.range(), members));
+#if defined(__clang__)
```

Comment:
These seems like a bug tied to a specific clang version?

Reply:
clang10 fails

Reply:
This is just going to cause errors in newer clang versions, isn't it?